database-plugins/src/test/java/co/cask/hydrator/plugin/db/batch/action/DBQueryActionTestRun.java

```diff
@@ -61,7 +61,10 @@ public void testAction() throws Exception {
       ImmutableMap.<String, String>builder()
         .put("connectionString", getConnectionURL())
         .put("jdbcPluginName", "hypersql")
-        .put("query", "delete from \"actionTest\" where day = '${logicalStartTime(yyyy-MM-dd,0m,UTC)}'")
+        .put("jdbcPluginType", "jdbc")
+        .put("query", "delete from \"actionTest\" where day = '1970-01-01'")
```

Comment:
If this isn't getting evaluated to 1970-01-01, it is a bug. Logical start time is being set to 0 (its an argument passed to workflowManager.start()), which is 1970-01-01.